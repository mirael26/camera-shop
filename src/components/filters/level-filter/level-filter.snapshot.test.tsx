import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import LevelFilter from './level-filter';

test('LevelFilter component renders correctly', () => {
  const {asFragment} = renderWithReduxAndRouter(<LevelFilter/>);
  expect(asFragment()).toMatchSnapshot();
});
