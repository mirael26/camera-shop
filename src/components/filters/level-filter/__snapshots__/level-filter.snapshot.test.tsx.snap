// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LevelFilter component renders correctly 1`] = `
<DocumentFragment>
  <fieldset
    class="catalog-filter__block"
  >
    <legend
      class="title title--h5"
    >
      Уровень
    </legend>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Нулевой"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Нулевой
        </span>
      </label>
    </div>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Любительский"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Любительский
        </span>
      </label>
    </div>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Профессиональный"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Профессиональный
        </span>
      </label>
    </div>
  </fieldset>
</DocumentFragment>
`;
